modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -9,6 +9,24 @@ The default installation of metering configures Hive to use an embedded Java dat
 
 There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
```

Comment:
Three

Reply:
The examples that follow this are for the yaml secret creation and the CLI method. Both of which have the 4 keys. Maybe configurations options as a term needs to change?

Reply:
I'm of the opinion that we should aim towards depreciating any examples that use the username and password configuration, in favor of the one that uses the `secretName` you're working on documented in this PR.